test/fixtures/nested_block_arguments.rb

```diff
@@ -0,0 +1,2 @@
+[[1, [2, 3]]].each do |one, (two, three)|
```

Comment:
Isn't this covered by `test/fixtures/double_nested_arguments.rb` as well?

Reply:
You're right, it is. I was capturing a few interesting cases and added them sequentially as I found them.